atomicfu/build.gradle

```diff
@@ -49,6 +49,10 @@ kotlin {
     // JVM -- always
     jvm()
 
+    wasm() {
+      d8()
```

Comment:
Why it is `d8` and not `nodejs`, like in the JS case?

Reply:
It does not matter really which engine we use in this case, d8 and nodejs both use v8 under the hood.

Reply:
Why not use the same engine then? It looks a bit confusing

Reply:
In wasm we are trying to use d8 because nodejs's v8 is always little outdated. But for here for kotlin 1.9 we can use nodejs as well. I can replace it with nodejs.